docs/resources/project_policy.md

```diff
@@ -0,0 +1,37 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "env0_project_policy Resource - terraform-provider-env0"
+subcategory: ""
+description: |-
+  
+---
+
+# env0_project_policy (Resource)
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **project_id** (String) id  of the project
+
+### Optional
+
+- **disable_destroy_environments** (Boolean) disable destroy environments
+- **include_cost_estimation** (Boolean) include cost estimation
+- **number_of_environments** (Number) number of environments per project, 0 indicates no limit
+- **number_of_environments_total** (Number) number of environments total, 0 indicates no limit
+- **requires_approval_default** (Boolean) requires approval
+- **skip_apply_when_plan_is_empty** (Boolean) skip apply when plan is empty
+- **skip_redundant_deployments** (Boolean) skip redundant deployments
```

Comment:
Same as the above.
In the `number_of_environments` and `number_of_environments_total` please also add `0 indicates no limit`

Reply:
e904bed2f63d25cc57f02fad28e9311306d29e4a